src/machine/machine_rp2350_flash.go

```diff
@@ -0,0 +1,111 @@
+//go:build rp2350
```

Comment:
Is this file a straight copy of `machine_rp2040_flash.go`? If so, rename `machine_rp2040_flash.go` to `machine_rp2_flash.go` and change its header to say `//go:build rp2040 || rp2350`.

Reply:
Oh yeah, will do